src/templates/footer.hbs

```diff
@@ -1,7 +1,58 @@
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/solutions/geoip2-enterprise-product-suite/anonymous-ip-database">GeoIP2 Anonymous IP Database</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/solutions/geoip2-enterprise-product-suite/enterprise-database">GeoIP2 Enterprise Database</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/geoip2-precision-services">GeoIP2 Precision Services</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/geoip2-databases">GeoIP2 Databases</a>
+        <a class="Footer-module--link" href="https://dev.maxmind.com/geoip/geoip2/geolite2">GeoLite2 Databases</a>
+      </div>
+      <div class="Footer-module--group Footer-module--group__support">
+        <h3 class="Footer-module--heading">Support</h3>
+        <a class="Footer-module--link" href="{{page_path 'help_center'}}">Support Center</a>
+        <a class="Footer-module--link" href="https://status.maxmind.com/">System Status</a>
+        <a class="Footer-module--link" href="{{page_path 'help_center'}}">Frequently Asked Questions</a>
+        <a class="Footer-module--link" href="{{page_path 'help_center'}}">GeoIP Data Correction Request</a><!-- To be updated with link to this form once it's migrated. -->
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/privacy-policy#additional-disclosures-for-california-residents">Do Not Sell My Personal Information</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/privacy-policy#california-notice-of-collection">Notice of Collection</a>
+      </div>
+      <div class="Footer-module--group Footer-module--group__developers">
+        <h3 class="Footer-module--heading">Developers</h3>
+        <a class="Footer-module--link" href="https://dev.maxmind.com/">Dev Center</a>
+        <a class="Footer-module--link" href="https://dev.maxmind.com/minfraud/">minFraud</a>
+        <a class="Footer-module--link" href="https://dev.maxmind.com/geoip/">GeoIP</a>
+      </div>
+      <div class="Footer-module--group Footer-module--group__company">
+        <h3 class="Footer-module--heading">Company</h3>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/company">About MaxMind</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/company/working-at-maxmind">Working at MaxMind</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/company/commitment-to-security">Commitment to Security</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/company/charitable-giving">Charitable Giving</a>
+        <a class="Footer-module--link" href="https://www.maxmind.com/en/company/contact-us">Contact Us</a>
+      </div>
+      <section class="Footer-module--copyright">
+        <p>© 2012-2021 MaxMind, Inc. All Rights Reserved. MaxMind, GeoIP, minFraud, and related trademarks belong to MaxMind, Inc.</p>
```

Comment:
Is there a way to replace `2021` with something dynamic that'll auto-increment?  Something serverside?

Reply:
Would something like this be ok?

`<script>document.write(new Date().getFullYear())</script> `

`<p>© 2012-<script>document.write(new Date().getFullYear())</script> MaxMind, Inc. All Rights Reserved. MaxMind, GeoIP, minFraud, and related trademarks belong to MaxMind, Inc.</p>`

Reply:
check this helper, dallas: https://developer.zendesk.com/documentation/help_center/help-center-templates/helpers/#date-helper